frontend/src/app/[locale]/events/[eventSlug]/surveys/[surveySlug]/page.tsx

```diff
@@ -68,8 +77,39 @@ export default async function SurveyPage({ params }: SurveyPageProps) {
   if (!survey) {
     notFound();
   }
-  const { form } = survey;
+  const {
+    form,
+    loginRequired,
+    anonymity,
+    maxResponsesPerUser,
+    countResponsesByCurrentUser,
+  } = survey;
   const { title, description, layout, fields } = form!;
+  const anonymityMessages =
+    translations.Survey.attributes.anonymity.secondPerson;
```

Comment:
You're already aliasing `translations.Survey` to `t` above, so
```suggestion
    t.attributes.anonymity.secondPerson;
```